package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
```

Comment:
![image](https://github.com/ANTOINE1128/To-Do-List/assets/93039359/9a2eff12-0213-46b9-94c0-79ca40c9a1e2)

**[OPTIONAL]**

- [ ] It would be nice if you make your commit message capitalized to make your pull request more professional 👍